src/VideoItem.jsx

```diff
@@ -0,0 +1,39 @@
+import React, {useState}from "react";
+import {VideoItems, Parag, TitleVideo, Secondary, IMG} from './components/Body.js';
+import Player from "./videoplayer.jsx";
+import { useHistory  } from "react-router-dom";
+
+function VideoItem({title, description,url,videoId, setTitles, setVideoList,searchResults}) {
+    
+    //const [results, setResults] = React.useState([]);
```

Comment:
I recommend removing commented code before the PR creation, so we are sure that we don't introduce commented code into our codebase